rskj-core/src/main/resources/reference.conf

```diff
@@ -314,6 +314,9 @@ rpc {
     callGasCap = 50000000
     # (experimental, OFF by default) RPC timeout in milliseconds. Disabled when it's zero.
     timeout = 0
+    # Maximum Response size allowed in bytes. If value is less than 0 then there is no limit.
+    maxResponseSize = -1
```

Comment:
I guess we should be consistent with other configs (eg. timeout) and use zero value to disable the feature, wdyt? and probably do not allow negative values?..

Reply:
Ok it has sense. I suppose no one would like to add a limit = 0 😓